### PR TITLE
Milestone 5: Refactor your code 

### PR DESCRIPTION
**Pull request by skabeo**
In this pull request;

- The main file was refactored to start with the user for the initial choice of options or to quit.
- A new `home_page.rb` file was created to handle the initial display to the user
- The `app.rb` file was refactored with a helper file `manage_person.rb`
- `manage_person.rb` handles the logic to create either a student or a teacher

### CODE REVIEW

File: src/app.rb [hunk 1]
```diff
@@ -1,140 +1,98 @@
...
-    puts 'Select which book you want to rent by entering its number'
+    if @books.empty?
+      puts 'No books created please create a book'
+      return
+    elsif @people.empty?
+      puts 'No people created please create a person'
+      return
+    end
+
+    puts 'Select a book from the following list of number'
```
Comment (Olamarx) [inline src/app.rb]: - [x] I will encourage you to try to make all requests for input and input validation placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input. By this, I mean all the requests from users should be independent of the create_book, create_rental. By this, the principle of [SOLID](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) would be met.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @skabeo,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Readme file professional ✔️
- Commit messages meaningful ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/app.rb (hunk 1 above)
Comment (Atifzada) [reply src/app.rb]: Thank you for the helpful review. We will come up with the solution

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @skabeo and @Atifzada 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**